mscore/instrwidget.cpp

```diff
@@ -56,7 +56,45 @@ void filterInstruments(QTreeWidget* instrumentList, const QString &searchPhrase)
             for (int cidx = 0; (ci = item->child(cidx)); ++cidx) {
                   // replace the unicode b (accidential) so a search phrase of "bb" would give Bb Trumpet...
                   QString text = ci->text(0).replace(QChar(0x266d), QChar('b'));
-                  bool isMatch = text.contains(searchPhrase, Qt::CaseInsensitive);
+
+                  // replace diacritics with regular latin characters
+                  // removes acute accents, diaeresis, cedilla, tilde, circumflex, etc.
+                  text = text.normalized(QString::NormalizationForm_KD);
+                  QString result;
+                  for (int i = 0; i < text.length(); ++i)  {
+                        if (text.at(i).category() != QChar::Mark_NonSpacing) {
+                              result.append(text.at(i));
+                              }
+                        }
+                  text = result.normalized(QString::NormalizationForm_C);
+
+                  // manually convert special characters like ae, ij, eszett
+                  QString manualReplaceDiacritic("");
+                  if (text.contains(QRegularExpression("[\\x{00F8}\\x{00DF}\\x{0133}\\x{00E6}]"))) {
+                        for (int i = 0; i < text.length(); ++i) {
+                              if (text.at(i) == QChar(0x00F8)) {
+                                    // o with stroke, &#248
+                                    manualReplaceDiacritic.append(QChar('o'));
+                                    }
+                              else if (text.at(i) == QChar(0x00DF)) {
+                                    // eszett, &#223
+                                    manualReplaceDiacritic.append(QString("ss"));
```

Comment:
indeed the correct transcription, even if it gets pronounced as "sz" or "sharp s"